packages/govuk-frontend/src/govuk/common/index.mjs

```diff
@@ -133,3 +133,17 @@ export function extractConfigByNamespace(configObject, namespace) {
   }
   return newObject
 }
+
+/**
+ * Checks if GOV.UK Frontend is supported on this page
+ *
+ * Some pages may be able to load our JavaScript, but GOV.UK Frontend may not be
+ * supported due to our check relying on support for `nomodule` rather than
+ * `type="module"` (as we can't detect the latter)
+ *
+ * @internal
+ * @returns {boolean} Whether GOV.UK Frontend is supported on this page
+ */
+export function isSupported() {
+  return document.body.classList.contains('govuk-frontend-supported')
```

Comment:
Possibly want to pass in `document` as a parameter here?

For use in Puppeteer helpers or Cheerio tests where `document` might not be global

Did something similar in Design System's [`new Navigation(document)`](https://github.com/alphagov/govuk-design-system/blob/e8224f2f1bbd4dae82c11b73b71495a07c120d93/src/javascripts/components/navigation.mjs#L8-L14)

Reply:
Ohhh, good spot. Missed a unit test for this method, which will require passing JSDom's document 😊 I'll add a `$scope` argument defaulted to `document.body` (to avoid people having to set it themselves).